Backend/app/rider-platform/rider-app/Main/spec/Storage/SharedRideConfigs.yaml

```
@@ -0,0 +1,69 @@
+    findByMerchantOperatingCityId:
+      kvFunction: findAllWithKV
+      where: merchantOperatingCityId
+
+    updateConfigValues:
+      kvFunction: updateOneWithKV
+      params: [pickupLocationSearchRadius, searchThresholdForSharedEstimate, searchRequestExpirySeconds, searchExpiryBufferSeconds, customerRemainingThresholdForFlowContinuation, dropLocationSearchRadius, actualPickupDistanceThreshold, actualDropDistanceThreshold, routeMatchingThreshold, geoHashPrecisionForRouteMatching, routeOverlapThreshold, updatedAt]
+      where: id
+
+    findAllByMerchantId:
+      kvFunction: findAllWithKV
+      where: merchantId
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)